### PR TITLE
🇺🇳 Add Language dropdown menu to navbar

### PR DESCRIPTION
**Pull request by lucianatl**
## Description
- added dropdown menu to top right corner of navbar
- english, spanish, french, chinese options
- cute little globe to the left of the dropdown

Fixes #<number>

## Screenshots
image

### CODE REVIEW

**Review by jackieo5023 — APPROVED** [review]
Looks good to me! Two things - there's a lint error that should be an easy fix, and could you create a new issue to add a language switching UI to mobile? (you could also potentially work on a simple design for this too 👀 I don't think this drop-down really fits for mobile use)

File: client/src/components/Navigation.jsx [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 import React, { useState } from 'react';
 import { NavLink } from 'react-router-dom';
-import { Layout, Drawer, Button, Menu } from 'antd';
-import { MenuOutlined } from '@ant-design/icons';
+import { Layout, Drawer, Button, Menu, Select } from 'antd';
+import { MenuOutlined, createFromIconfontCN } from '@ant-design/icons';
```
Comment (n3a9) [inline client/src/components/Navigation.jsx:6]: If we're not using `createFromIconfontCN` maybe remove from PR just to avoid the lint warning.

**Review by n3a9 — COMMENTED** [review]
Hm, not sure what's going on but for the [deployed version of this branch](https://nawc-git-language-dropdown-hack4impact1.vercel.app/) it seems to be broken for me.

**Chrome**

Xnip2021-03-08_16-23-46


**Firefox**

Xnip2021-03-08_16-23-53


**Safari**

Xnip2021-03-08_16-24-02

**Comment by jackieo5023** [comment]
Thanks for catching that Neeraj 🤦 I think we just need to add the svg to git (`git add .`)

**Review by Lam7150 — APPROVED** [review]
Looks good!

**Pull request merged**